tools/test

```diff
@@ -74,7 +74,7 @@ fail=0
 
 if [ -z "$TESTDIRS" ]; then
     # NB: Relies on paths being prefixed with './'.
-    TESTDIRS=($(git ls-files -- '*_test.go' | grep -vE '^(vendor|prog|experimental)/' | xargs -n1 dirname | sort -u | sed -e 's|^|./|'))
+    TESTDIRS=($(git ls-files -- '*_test.go' | grep -vE '^(vendor|experimental)/' | xargs -n1 dirname | sort -u | sed -e 's|^|./|'))
```

Comment:
I stumbled over this: https://github.com/weaveworks/build-tools/pull/103/files#diff-098f6bcd4621d373cade4e832627b4f6R81

Is this sufficient?